docs/diagrams/DetailedTaskClassDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Package Model <<Rectangle>>{
+
+Abstract Class Task
```

Comment:
The abstract keyword for Task is missing from the diagram?

Reply:
The diagram is actually already updated. Abstract classes in PlantUML are represented by italicizing the name instead of using the `{abstract}` notation. Italicizing is still in line with our project requirements though!